registry/donor/models.py

```diff
@@ -99,6 +101,111 @@ class DonorsOverview(db.Model):
     def __repr__(self):
         return f"<DonorsOverview ({self.rodne_cislo})>"
 
+    @classmethod
+    def refresh_override(cls, commit=True):
+        if sqlite3.sqlite_version_info[0] >= 3 and sqlite3.sqlite_version_info[1] >= 33:
```

Comment:
You can also compare tuples:
```suggestion
        if sqlite3.sqlite_version_info >= (3, 33):
```

Reply:
These two queries should work for now and the plan is to merge them into the main refresh one.